### PR TITLE
Links in navigation menu (still with router)

### PR DESCRIPTION
**Pull request by Saacy**
#96

### CODE REVIEW

**Comment by MrBurrBurr** [comment]
Looks good to me, lets see what @Aareksio says.

File: src/components/NavigationLink.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<router-link tag="div" :to="to" class="navigation-link" active-class="navigation-link--active" :exact="true">
+	<router-link :to="to" :href="to.name" class="navigation-link" active-class="navigation-link--active" :exact="true">
```
Comment (Aareksio) [inline src/components/NavigationLink.vue]: Use `$router.resolve(to).href` for `href`

**Comment by MrBurrBurr** [comment]
Closed for #101

**Comment by Aareksio** [comment]
One feature at a time please :)

**Pull request closed without merging**